src/gluonts/mx/trainer/_base.py

```diff
@@ -126,6 +129,7 @@ def __init__(
             AveragingStrategy, IterationAveragingStrategy
         ] = SelectNBestMean(num_models=1),
         post_initialize_cb: Optional[Callable[[mx.gluon.Block], None]] = None,
+        post_epoch_callback: Optional[Callable[[int, float], None]] = None,
```

Comment:
I think your naming (`_callback`) is better than the one for the callback which was there already (`_cb`). Maybe you could uniform both using the `_callback` naming, as part of this PR?